src/data/faq.json

```
@@ -316,7 +316,13 @@
                         "anchor": "minimum_requirements",
                         "active": true,
                         "textblock": [
-                            "<b>UPDATE</b><br/><a href='https://www.coronawarn.app/en/blog/2020-12-15-ios-12-5' target='_blank' rel='noopener noreferrer'>Telekom and SAP to make Corona Warn app available for iOS 12.5.</a><hr/>",
+                            "<b>UPDATE February 10, 2021</b>",
+                            "<b>iOS - Minimum operating system requirements</b>",
+                            "The minimum operating system level for iPhone 5s, iPhone 6 and iPhone 6 Plus devices is iOS 12.5. For iPhone SE (1st generation), iPhone 6s, iPhone 6s Plus and newer iOS devices the minimum is iOS 13.7. These versions of the iOS operating system include the necessary v2 version of the Apple Exposure Notification Framework.",

```

I would remove the v from the 2 since v stands for version. 

```suggestion
                            "The minimum operating system level for iPhone 5s, iPhone 6 and iPhone 6 Plus devices is iOS 12.5. For iPhone SE (1st generation), iPhone 6s, iPhone 6s Plus and newer iOS devices the minimum is iOS 13.7. These versions of the iOS operating system include the necessary version 2 of the Apple Exposure Notification Framework.",
```